unit_test/include/apply_1q_gate_test.hpp

```diff
@@ -203,7 +203,7 @@ TEST_F(Apply1QGateTest, RotationXY)
   psi.ApplyRotationZ(qubit, -phi);
   psi.ApplyRotationX(qubit, -theta);
   psi.ApplyRotationZ(qubit,  phi);
-  ASSERT_TRUE( psi.ComputeOverlap(psi_0).real() < 1.-accepted_error_);
+  ASSERT_TRUE( psi.ComputeOverlap(psi_0).real() > 1.-accepted_error_);
```

Comment:
Here and below, can we use abs() to flag the unwanted increase in the overlap too?